kernel/include/ext2.h

```diff
@@ -143,7 +143,10 @@ struct ext2_inodetable {
 	uint32_t blocks;
 	uint32_t flags;
 	uint32_t osd1;
-	uint32_t block[15];
+	union {
```

Comment:
No anonymous unions in C99. I took the unfortunately ugly approach of leaving this as just `block` and using a macro to get the pseudo-`symlink` member, which meant I had to do even more terrible things where you were taking `sizeof(inode->symlink)`, and I am not proud of this :(